services/deps-rs/deps-rs-repo.service.js

```diff
@@ -0,0 +1,38 @@
+import { pathParams } from '../index.js'
+import { BaseDepsRsService, description } from './deps-rs-base.js'
+
+export default class DepsRsRepo extends BaseDepsRsService {
+  static category = 'dependencies'
+  static route = { base: 'deps-rs/repo', pattern: ':site/:user/:repo' }
+
+  static openApi = {
+    '/deps-rs/repo/{site}/{user}/{repo}': {
+      get: {
+        summary: 'Deps.rs Repository Dependencies',
+        description,
+        parameters: pathParams(
+          {
+            name: 'site',
+            example: 'github',
+          },
```

Comment:
Is there a list of supported platforms for this? If so, can we make it an enum rather than free text?
You can see coveralls for an example of this:
https://github.com/badges/shields/blob/cb0af9dd01a05e802505ace38cd083737556d25f/services/coveralls/coveralls.service.js#L17-L30